renovate-config-iac.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Shared Renovate configuration for dsb-norge IaC projects where major updates are skipped",
```

Comment:
Are majors skipped? The config further down says major.enabled = true 🤔

Reply:
nice catch. forgot to update description.